src/Sulu/Component/Rest/ListBuilder/AbstractListBuilder.php

```diff
@@ -238,8 +238,19 @@ public function search($search)
      */
     public function sort(AbstractFieldDescriptor $fieldDescriptor, $order = self::SORTORDER_ASC)
     {
-        $this->sortField = $fieldDescriptor;
-        $this->sortOrder = $order;
+        $this->sortFields = array($fieldDescriptor);
```

Comment:
IMO it also doesn't make sense to override this behavior... Do we have any code relying on this?